docs/docusaurus/versioned_docs/version-0.18/cloud/about_gx.md

```diff
@@ -57,34 +51,21 @@ The GX Agent is typically deployed in your organization's deployment environment
 
 GX Cloud deployments can be tailored to meet your specific business requirements. To determine which deployment pattern and GX architectural components are most suitable for your organization, determine how your data is accessed and how users interact with GX Cloud.
 
-The following table shows the connection options.
-
-| Deployment pattern | Environment | GX Agent | GX OSS client |
-| :-- | :-- | :--: | :--: |
-| Org-hosted | Deployment | <div align="center"><img alt="Yes" src="/img/simple_orange_circle.svg"/></div> | |
-| Self-hosted | Local | <div align="center"><img alt="Yes" src="/img/simple_orange_circle.svg"/></div> | |
-| Orchestrated | Deployment or local | | <div align="center"><img alt="Yes" src="/img/simple_orange_circle.svg"/></div> |
-
-The following table shows how users can access GX Cloud.
-
-| Access method | GX Cloud | GX OSS client |
-| :-- | :--: | :--: |
-| UI | <div align="center"><img alt="Yes" src="/img/simple_orange_circle.svg"/></div> | |
-| Python API | | <div align="center"><img alt="Yes" src="/img/simple_orange_circle.svg"/></div>  |
-| UI and Python API | <div align="center"><img alt="Yes" src="/img/simple_orange_circle.svg"/></div> | <div align="center"><img alt="Yes" src="/img/simple_orange_circle.svg"/></div> |
-
 
 ### Org-hosted deployment pattern
+
 In an org-hosted deployment, you run the GX Agent in your organization's deployment environment, for example, development or production. The GX Agent is an intermediary between GX Cloud and your organization's deployment environment data stores. Organization users can interact with GX Cloud using the web UI, the GX OSS client, or both.
```

Comment:
```suggestion
In an org-hosted deployment, you run the GX Agent in your organization's deployment environment. The GX Agent is an intermediary between GX Cloud and your organization's deployment environment data stores. Organization users can interact with GX Cloud using the web UI, the GX OSS client, or both.
```
The interjected example doesn't really feel necessary and having so many commas in one sentence makes it a bit less readable.